Rekey, and freshen build instructions for running on Ubuntu 18.04

abliss opened the pull request:
Adds a few new dependencies to the apt-get command.

Replaces /bin/sh in shebang line with /bin/bash
(Ubuntu used to have /bin/sh point to bash, but now it points to dash:
  https://wiki.ubuntu.com/DashAsBinSh
The shell scripts use the "source" bash builtin which is missing from dash.)

Busybox apparently used to install in /usr/bin but now it's in /bin.

Commit:
Update files list.

dwrensha committed:
Switch to vagrant-spk.

dwrensha committed:
Use the system's gitweb files.

dwrensha committed:
Cleanup.

dwrensha committed:
note about vagrant-spk

dwrensha committed:
markdown...

dwrensha committed:
Markdown?!

neynah committed:
Graphics for app market and new ui

Commit:
Add some metadata.

dwrensha committed:
Merge pull request #2 from neynah/master

Graphics for app market and new ui

Commit:
Add more metadata

Commit:
Bump version to 0.0.6

Commit:
Update to a new-style shortDescription.

Commit:
Bump to version 7.

Commit:
vagrant-spk updates

Commit:
New release using new sandstorm-http-bridge with flow control.

Commit:
v0.0.9

Commit:
add MIT license

abliss commented:
These are the changes I had to make locally to get the spk building on my Ubuntu 18.04 machine. Feel free to merge any bits you want.

ocdtrekkie commented:
Do we have any word on if @paulproteus can offer up the original key, add you as a collaborator, and/or transfer ownership of the repository?

abliss committed:
Freshen for modern vagrant-spk and debian distro.

Also replaces 'unlink' with 'rm -rf' to make it easier to reprovision.

abliss committed:
Add optional static publishing.

This merges in some of the technology of paulproteus's fork, but in a less
intrusive manner. Once the user clicks on the "publish" button, a post-receive
hook is permanently installed which copies the contents of the master branch to
the grain's public id.

abliss committed:
Rekey and rev version.

abliss committed:
Fix publish button.

Previously, it enabled publish-on-push, but did not actually publish what was
already on master.
